internal/defaults/defaults.go

```diff
@@ -21,12 +21,12 @@ const (
 
 	HubbleCAGenerate         = false
 	HubbleCAReuseSecret      = false
-	HubbleCACommonName       = "hubble-ca.cilium.io"
+	HubbleCACommonName       = "Cilium CA"
 	HubbleCAValidityDuration = 3 * 365 * 24 * time.Hour
-	HubbleCASecretName       = "hubble-ca-secret"
+	HubbleCASecretName       = "cilium-ca"
 
 	HubbleCAConfigMapCreate = false
-	HubbleCAConfigMapName   = "hubble-ca-cert"
+	HubbleCAConfigMapName   = "cilium-ca-cert"
```

Comment:
The config map has been deprecated in v1.10 and v1.11 will remove it completely (see https://github.com/cilium/cilium/pull/17294). Instead of renaming it, we should consider removing it instead. cc @gandro

Reply:
Agreed that we should remove this. Doesn't have to be part of this PR though, unless if it helps with the transition.

Reply:
Opened https://github.com/cilium/certgen/pull/43 to deprecate the hubble CA ConfigMap.